configuration/debian/tedge_agent/preinst

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
This pre-installation script was added to make sure that all stale directories created with `root` permissions using the older debian package are cleaned up properly before installing this version.

Reply:
This is a way to "cleanup" the mess from the previous version? Is it temporary? How long do we intend to keep this here?

Reply:
I'd say that we'd need this script in the long term as well, so that we can control the desired state of an environment before installation instead of relying on the `postrm` script of an older installation that may or may not clean things that way the newer version expects it to.